unicodetools/data/linkification/dev/LinkEmail.txt

```diff
@@ -0,0 +1,1298 @@
+# LinkEmail.txt
+# Date: 2025-12-14, 06:39:23 GMT 
+# © 2025 Unicode®, Inc.
+# Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
+# For terms of use and license, see https://www.unicode.org/terms_of_use.html
+#
+# The usage and stability of these values is covered in https://www.unicode.org/reports/tr58/
+#
+# ================================================
+#
+# Property:	Link_Email
+# Format
+#
+# Field 0: code point range
+# Field 1: binary value
+#          For more information, see https://www.unicode.org/reports/tr58/#property-data. 
+#
+# For the purpose of link detection and formatting operations, the property Link_Email is defined as
+# a binary property.
+#
+# The short name of the property is the same as its long name.
+#
+#  All code points not explicitly listed for Link_Email
+#  have the value No.
+#
+# @missing: 0000..10FFFF; No
```

Comment:
same as above -- we haven't written `@missing` lines for binary properties; they just all default to `No`

of course the `@missing` line, when present in a data file, should have the same syntax as the value lines in the same file. so if we explicitly print `Yes` then i guess we should have this line as well. but i think it just shows even more how weird it is. i think we should omit the `Yes` values and also omit the `@missing` line.